internal/pkg/aws/handlers/archer_user_agent.go

```diff
@@ -0,0 +1,26 @@
+// Copyright 2019 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+// SPDX-License-Identifier: Apache-2.0
+
+package handlers
+
+import (
+	"fmt"
+	"runtime"
+
+	"github.com/aws/amazon-ecs-cli-v2/internal/pkg/version"
+	"github.com/aws/aws-sdk-go/aws/request"
+)
+
+const UserAgentHeader = "Archer-User-Agent"
+
+// ArcherUserAgentHandler returns a http request handler that sets a custom user agent to all aws requests
+func ArcherUserAgentHandler() request.NamedHandler {
+	return request.NamedHandler{
+		Name: "ArcherUserAgent",
```

Comment:
Unimportant Nit: this could be "ArcherUserAgentHandler", because its a handler, not the actual user agent 
(that also follows the pattern in our other tools)

Reply:
Addressed